<?xml version='1.0' encoding='utf-8'?>
<robot name="husky_ur10">
  <material name="dark_grey">
    <color rgba="0.2 0.2 0.2 1.0" />
  </material>
  <material name="medium_grey">
    <color rgba="0.6 0.6 0.6 1.0" />
  </material>
  <material name="light_grey">
    <color rgba="0.8 0.8 0.8 1.0" />
  </material>
  <material name="yellow">
    <color rgba="0.8 0.8 0.0 1.0" />
  </material>
  <material name="black">
    <color rgba="0.15 0.15 0.15 1.0" />
  </material>
  <material name="white">
    <color rgba="1.0 1.0 1.0 1.0" />
  </material>
  <material name="red">
    <color rgba="1.0 0.0 0.0 1.0" />
  </material>
  
  <link name="husky_base_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/base_link.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0.0 0 0.061875" />
      <geometry>
        <box size="0.9874 0.5709 0.12375" />
      </geometry>
    </collision>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0.17562499999999998" />
      <geometry>
        <box size="0.7899200000000001 0.5709 0.10375" />
      </geometry>
    </collision>
  </link>
  
  <link name="husky_base_footprint" />
  <joint name="husky_base_footprint_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 -0.13228" />
    <parent link="husky_base_link" />
    <child link="husky_base_footprint" />
  </joint>
  
  <link name="husky_inertial_link">
    <inertial>
      <mass value="46.034" />
      <origin xyz="-0.00065 -0.085 0.062" />
      <inertia ixx="0.6022" ixy="-0.02364" ixz="-0.1197" iyy="1.7386" iyz="-0.001544" izz="2.0296" />
    </inertial>
  </link>
  <joint name="husky_inertial_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 0" />
    <parent link="husky_base_link" />
    <child link="husky_inertial_link" />
  </joint>
  
  
  <link name="husky_front_left_wheel_link">
    <inertial>
      <mass value="2.637" />
      <origin xyz="0 0 0" />
      <inertia ixx="0.02467" ixy="0" ixz="0" iyy="0.04411" iyz="0" izz="0.02467" />
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/wheel.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="1.570795 0 0" xyz="0 0 0" />
      <geometry>
        <cylinder length="0.1143" radius="0.1651" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="husky_front_left_wheel_link">
    <mu1 value="1.0" />
    <mu2 value="1.0" />
    <kp value="10000000.0" />
    <kd value="1.0" />
    <fdir1 value="1 0 0" />
  </gazebo>
  <joint name="husky_front_left_wheel" type="continuous">
    <parent link="husky_base_link" />
    <child link="husky_front_left_wheel_link" />
    <origin rpy="0 0 0" xyz="0.256 0.2854 0.03282" />
    <axis xyz="0 1 0" />
  </joint>
  <link name="husky_front_right_wheel_link">
    <inertial>
      <mass value="2.637" />
      <origin xyz="0 0 0" />
      <inertia ixx="0.02467" ixy="0" ixz="0" iyy="0.04411" iyz="0" izz="0.02467" />
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/wheel.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="1.570795 0 0" xyz="0 0 0" />
      <geometry>
        <cylinder length="0.1143" radius="0.1651" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="husky_front_right_wheel_link">
    <mu1 value="1.0" />
    <mu2 value="1.0" />
    <kp value="10000000.0" />
    <kd value="1.0" />
    <fdir1 value="1 0 0" />
  </gazebo>
  <joint name="husky_front_right_wheel" type="continuous">
    <parent link="husky_base_link" />
    <child link="husky_front_right_wheel_link" />
    <origin rpy="0 0 0" xyz="0.256 -0.2854 0.03282" />
    <axis xyz="0 1 0" />
  </joint>
  <link name="husky_rear_left_wheel_link">
    <inertial>
      <mass value="2.637" />
      <origin xyz="0 0 0" />
      <inertia ixx="0.02467" ixy="0" ixz="0" iyy="0.04411" iyz="0" izz="0.02467" />
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/wheel.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="1.570795 0 0" xyz="0 0 0" />
      <geometry>
        <cylinder length="0.1143" radius="0.1651" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="rear_left_wheel_link">
    <mu1 value="1.0" />
    <mu2 value="1.0" />
    <kp value="10000000.0" />
    <kd value="1.0" />
    <fdir1 value="1 0 0" />
  </gazebo>
  <joint name="husky_rear_left_wheel" type="continuous">
    <parent link="husky_base_link" />
    <child link="husky_rear_left_wheel_link" />
    <origin rpy="0 0 0" xyz="-0.256 0.2854 0.03282" />
    <axis xyz="0 1 0" />
  </joint>
  <link name="husky_rear_right_wheel_link">
    <inertial>
      <mass value="2.637" />
      <origin xyz="0 0 0" />
      <inertia ixx="0.02467" ixy="0" ixz="0" iyy="0.04411" iyz="0" izz="0.02467" />
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/wheel.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="1.570795 0 0" xyz="0 0 0" />
      <geometry>
        <cylinder length="0.1143" radius="0.1651" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="rear_right_wheel_link">
    <mu1 value="1.0" />
    <mu2 value="1.0" />
    <kp value="10000000.0" />
    <kd value="1.0" />
    <fdir1 value="1 0 0" />
  </gazebo>
  <joint name="husky_rear_right_wheel" type="continuous">
    <parent link="husky_base_link" />
    <child link="husky_rear_right_wheel_link" />
    <origin rpy="0 0 0" xyz="-0.256 -0.2854 0.03282" />
    <axis xyz="0 1 0" />
  </joint>
  <link name="husky_top_chassis_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/top_chassis.dae" />
      </geometry>
    </visual>
  </link>
  
  <joint name="husky_top_chassis_joint" type="fixed">
    <parent link="husky_base_link" />
    <child link="husky_top_chassis_link" />
  <origin xyz="0 0 0" rpy="0 0 0" /></joint>
  <link name="husky_front_bumper_link">
    <visual>
      <geometry>
        <mesh filename="package://husky_description/meshes/bumper.dae" />
      </geometry>
    </visual>
  </link>
  
  <joint name="husky_front_bumper" type="fixed">
    <origin rpy="0 0 0" xyz="0.48 0 0.091" />
    <parent link="husky_base_link" />
    <child link="husky_front_bumper_link" />
  </joint>
  <link name="husky_rear_bumper_link">
    <visual>
      <geometry>
        <mesh filename="package://husky_description/meshes/bumper.dae" />
      </geometry>
    </visual>
  </link>
  
  <joint name="husky_rear_bumper" type="fixed">
    <origin rpy="0 0 3.14159" xyz="-0.48 0 0.091" />
    <parent link="husky_base_link" />
    <child link="husky_rear_bumper_link" />
  </joint>
  
  <link name="husky_user_rail_link">
    <visual>
      <geometry>
        <mesh filename="package://husky_description/meshes/user_rail.dae" />
      </geometry>
    </visual>
  </link>
  
  <joint name="husky_user_rail" type="fixed">
    <origin rpy="0 0 0" xyz="0.272 0 0.245" />
    <parent link="husky_base_link" />
    <child link="husky_user_rail_link" />
  </joint>
  
  <link name="husky_top_plate_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/top_plate.dae" />
      </geometry>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://husky_description/meshes/top_plate.stl" />
      </geometry>
    </collision>
  </link>
  
  <joint name="husky_top_plate_joint" type="fixed">
    <parent link="husky_base_link" />
    <child link="husky_top_plate_link" />
    <origin rpy="0 0 0" xyz="0.0812 0 0.245" />
  </joint>
  
  <link name="husky_top_plate_front_link" />
  <joint name="husky_top_plate_front_joint" type="fixed">
    <parent link="husky_top_plate_link" />
    <child link="husky_top_plate_front_link" />
    <origin rpy="0 0 0" xyz="0.36367 0 0.00639" />
  </joint>
  
  <link name="husky_top_plate_rear_link" />
  <joint name="husky_top_plate_rear_joint" type="fixed">
    <parent link="husky_top_plate_link" />
    <child link="husky_top_plate_rear_link" />
    <origin rpy="0 0 0" xyz="-0.36633 0 0.00639" />
  </joint>
  
  <link name="husky_imu_link" />
  <joint name="husky_imu_joint" type="fixed">
    <origin rpy="0 -1.5708 3.1416" xyz="0.19 0 0.149" />
    <parent link="husky_base_link" />
    <child link="husky_imu_link" />
  </joint>
  <gazebo reference="imu_link">
  </gazebo>
  <gazebo>
    <plugin filename="libgazebo_ros_control.so" name="gazebo_ros_control">
      <robotNamespace>/</robotNamespace>
      <legacyModeNS>true</legacyModeNS>
    </plugin>
  </gazebo>
  <gazebo>
    <plugin filename="libhector_gazebo_ros_imu.so" name="imu_controller">
      <robotNamespace>/</robotNamespace>
      <updateRate>50.0</updateRate>
      <bodyName>husky_base_link</bodyName>
      <topicName>imu/data</topicName>
      <accelDrift>0.005 0.005 0.005</accelDrift>
      <accelGaussianNoise>0.005 0.005 0.005</accelGaussianNoise>
      <rateDrift>0.005 0.005 0.005 </rateDrift>
      <rateGaussianNoise>0.005 0.005 0.005 </rateGaussianNoise>
      <headingDrift>0.005</headingDrift>
      <headingGaussianNoise>0.005</headingGaussianNoise>
    </plugin>
  </gazebo>
  <gazebo>
    <plugin filename="libhector_gazebo_ros_gps.so" name="gps_controller">
      <robotNamespace>/</robotNamespace>
      <updateRate>40</updateRate>
      <bodyName>husky_base_link</bodyName>
      <frameId>husky_base_link</frameId>
      <topicName>navsat/fix</topicName>
      <velocityTopicName>navsat/vel</velocityTopicName>
      <referenceLatitude>49.9</referenceLatitude>
      <referenceLongitude>8.9</referenceLongitude>
      <referenceHeading>90</referenceHeading>
      <referenceAltitude>0</referenceAltitude>
      <drift>0.0001 0.0001 0.0001</drift>
    </plugin>
  </gazebo>


  
  
  
  

  
  <link name="ur10_base_link" />
  <link name="ur10_base_link_inertia">
    <visual>
      <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/base.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/base.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="4.0" />
      <origin rpy="0 0 0" xyz="0 0 0" />
      <inertia ixx="0.0061063308908" ixy="0.0" ixz="0.0" iyy="0.0061063308908" iyz="0.0" izz="0.01125" />
    </inertial>
  </link>
  <link name="ur10_shoulder_link">
    <visual>
      <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/shoulder.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/shoulder.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="7.778" />
      <origin rpy="0 0 0" xyz="0 0 0" />
      <inertia ixx="0.03147431257693659" ixy="0.0" ixz="0.0" iyy="0.03147431257693659" iyz="0.0" izz="0.021875624999999996" />
    </inertial>
  </link>
  <link name="ur10_upper_arm_link">
    <visual>
      <origin rpy="1.5707963267948966 0 -1.5707963267948966" xyz="0 0 0.220941" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/upperarm.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="1.5707963267948966 0 -1.5707963267948966" xyz="0 0 0.220941" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/upperarm.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="12.93" />
      <origin rpy="0 1.5707963267948966 0" xyz="-0.306 0.0 0.175" />
      <inertia ixx="0.42175380379841093" ixy="0.0" ixz="0.0" iyy="0.42175380379841093" iyz="0.0" izz="0.03636562499999999" />
    </inertial>
  </link>
  <link name="ur10_forearm_link">
    <visual>
      <origin rpy="1.5707963267948966 0 -1.5707963267948966" xyz="0 0 0.049042" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/forearm.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="1.5707963267948966 0 -1.5707963267948966" xyz="0 0 0.049042" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/forearm.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="3.87" />
      <origin rpy="0 1.5707963267948966 0" xyz="-0.28615 0.0 0.049042" />
      <inertia ixx="0.11106969409710458" ixy="0.0" ixz="0.0" iyy="0.11106969409710458" iyz="0.0" izz="0.010884375" />
    </inertial>
  </link>
  <link name="ur10_wrist_1_link">
    <visual>
      
      <origin rpy="1.5707963267948966 0 0" xyz="0 0 -0.1149" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/wrist1.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="1.5707963267948966 0 0" xyz="0 0 -0.1149" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/wrist1.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="1.96" />
      <origin rpy="0 0 0" xyz="0 0 0" />
      <inertia ixx="0.005108247956699999" ixy="0.0" ixz="0.0" iyy="0.005108247956699999" iyz="0.0" izz="0.005512499999999999" />
    </inertial>
  </link>
  <link name="ur10_wrist_2_link">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 -0.1158" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/wrist2.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 -0.1158" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/wrist2.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="1.96" />
      <origin rpy="0 0 0" xyz="0 0 0" />
      <inertia ixx="0.0040792483683" ixy="0.0" ixz="0.0" iyy="0.0040792483683" iyz="0.0" izz="0.005512499999999999" />
    </inertial>
  </link>
  <link name="ur10_wrist_3_link">
    <visual>
      <origin rpy="1.5707963267948966 0 0" xyz="0 0 -0.0922" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/visual/wrist3.dae" />
      </geometry>
      <material name="LightGrey">
        <color rgba="0.7 0.7 0.7 1.0" />
      </material>
    </visual>
    <collision>
      <origin rpy="1.5707963267948966 0 0" xyz="0 0 -0.0922" />
      <geometry>
        <mesh filename="package://ur_description/meshes/ur10/collision/wrist3.stl" />
      </geometry>
    </collision>
    <inertial>
      <mass value="0.202" />
      <origin rpy="0 0 0" xyz="0.0 0.0 -0.01525" />
      <inertia ixx="0.00011792166116465" ixy="0.0" ixz="0.0" iyy="0.00011792166116465" iyz="0.0" izz="0.00020452500000000002" />
    </inertial>
  </link>
  
  
      
  <joint name="ur10_husky_to_ur10" type="fixed">
    <parent link="husky_base_link" />
    
    <child link="ur10_base_link" />
    
    <origin rpy="0 0 0" xyz="0 0 0.2" />
    
  </joint>
  
  
  
  <joint name="ur10_base_link-base_link_inertia" type="fixed">
    <parent link="ur10_base_link" />
    <child link="ur10_base_link_inertia" />
    
    <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
  </joint>
  <joint name="ur10_shoulder_pan_joint" type="continuous">
    <parent link="ur10_base_link_inertia" />
    <child link="ur10_shoulder_link" />
    <origin rpy="0 0 0" xyz="0 0 0.1273" />
    <axis xyz="0 0 1" />
    <limit effort="330.0" lower="-6.283185307179586" upper="6.283185307179586" velocity="2.0943951023931953" />
    <dynamics damping="0" friction="0" />
  </joint>
  <joint name="ur10_shoulder_lift_joint" type="continuous">
    <parent link="ur10_shoulder_link" />
    <child link="ur10_upper_arm_link" />
    <origin rpy="1.570796327 0 0" xyz="0 0 0" />
    <axis xyz="0 0 1" />
    <limit effort="330.0" lower="-6.283185307179586" upper="6.283185307179586" velocity="2.0943951023931953" />
    <dynamics damping="0" friction="0" />
  </joint>
  <joint name="ur10_elbow_joint" type="continuous">
    <parent link="ur10_upper_arm_link" />
    <child link="ur10_forearm_link" />
    <origin rpy="0 0 0" xyz="-0.612 0 0" />
    <axis xyz="0 0 1" />
    <limit effort="150.0" lower="-3.141592653589793" upper="3.141592653589793" velocity="3.141592653589793" />
    <dynamics damping="0" friction="0" />
  </joint>
  <joint name="ur10_wrist_1_joint" type="continuous">
    <parent link="ur10_forearm_link" />
    <child link="ur10_wrist_1_link" />
    <origin rpy="0 0 0" xyz="-0.5723 0 0.163941" />
    <axis xyz="0 0 1" />
    <limit effort="56.0" lower="-6.283185307179586" upper="6.283185307179586" velocity="3.141592653589793" />
    <dynamics damping="0" friction="0" />
  </joint>
  <joint name="ur10_wrist_2_joint" type="continuous">
    <parent link="ur10_wrist_1_link" />
    <child link="ur10_wrist_2_link" />
    <origin rpy="1.570796327 0 0" xyz="0 -0.1157 -2.373046667922381e-11" />
    <axis xyz="0 0 1" />
    <limit effort="56.0" lower="-6.283185307179586" upper="6.283185307179586" velocity="3.141592653589793" />
    <dynamics damping="0" friction="0" />
  </joint>
  <joint name="ur10_wrist_3_joint" type="continuous">
    <parent link="ur10_wrist_2_link" />
    <child link="ur10_wrist_3_link" />
    <origin rpy="1.570796326589793 3.141592653589793 3.141592653589793" xyz="0 0.0922 -1.891053610911353e-11" />
    <axis xyz="0 0 1" />
    <limit effort="56.0" lower="-6.283185307179586" upper="6.283185307179586" velocity="3.141592653589793" />
    <dynamics damping="0" friction="0" />
  </joint>
  
  <link name="ur10_base" />
  <joint name="ur10_base_link-base_fixed_joint" type="fixed">
    
    <origin rpy="0 0 3.141592653589793" xyz="0 0 0" />
    <parent link="ur10_base_link" />
    <child link="ur10_base" />
  </joint>
  
  <link name="ur10_flange" />
  <joint name="ur10_wrist_3-flange" type="continuous">
    <parent link="ur10_wrist_3_link" />
    <child link="ur10_flange" />
    <origin rpy="0 -1.5707963267948966 -1.5707963267948966" xyz="0 0 0" />
  <axis xyz="0 0 1" /></joint>
  
  <link name="ur10_tool0" />
  <joint name="ur10_flange-tool0" type="fixed">
    
    <origin rpy="1.5707963267948966 0 1.5707963267948966" xyz="0 0 0" />
    <parent link="ur10_flange" />
    <child link="ur10_tool0" />
  </joint>

<transmission name="husky_front_left_wheel_trans"><type>transmission_interface/SimpleTransmission</type><joint name="husky_front_left_wheel"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="husky_front_left_wheel_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="husky_front_right_wheel_trans"><type>transmission_interface/SimpleTransmission</type><joint name="husky_front_right_wheel"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="husky_front_right_wheel_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="husky_rear_left_wheel_trans"><type>transmission_interface/SimpleTransmission</type><joint name="husky_rear_left_wheel"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="husky_rear_left_wheel_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="husky_rear_right_wheel_trans"><type>transmission_interface/SimpleTransmission</type><joint name="husky_rear_right_wheel"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="husky_rear_right_wheel_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_shoulder_pan_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_shoulder_pan"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_shoulder_pan_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_shoulder_lift_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_shoulder_lift"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_shoulder_lift_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_elbow_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_elbow"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_elbow_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_wrist_1_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_wrist_1"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_wrist_1_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_wrist_2_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_wrist_2"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_wrist_2_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission><transmission name="ur10_wrist_3_trans"><type>transmission_interface/SimpleTransmission</type><joint name="ur10_wrist_3"><hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface></joint><actuator name="ur10_wrist_3_trans_actuator"><mechanicalReduction>1.0</mechanicalReduction></actuator></transmission></robot>
